docs/syntax/flowchart.md

```diff
@@ -926,6 +926,66 @@ flowchart TD
     A@{ shape: tag-rect, label: "Tagged process" }
 ```
 
+## Special shapes in Mermaid Flowcharts (v11.3.0+)
+
+Mermaid also introduces 2 special shapes to enhance your flowcharts: **icon** and **image**. These shapes allow you to include icons and images directly within your flowcharts, providing more visual context and clarity.
+
+### Icon Shape
+
+You can use the `icon` shape to include an icon in your flowchart. To use icons, you need to register the icon pack first. Follow the instructions provided [here](https://mermaid.js.org/syntax/architecture.html#icons). The syntax for defining an icon shape is as follows:
```

Comment:
We should move the icon document out of syntax/architecture, not that it's available in other diagrams.